apps/verticalhome/test/marionette/grid_layout_test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var assert = require('assert');
+
+marionette('Vertical - grid', function() {
+
+  var client = marionette.client(require(__dirname + '/client_options.js'));
+  var actions, home, system;
+
+  setup(function() {
+    actions = client.loader.getActions();
+    home = client.loader.getAppClass('verticalhome');
+    system = client.loader.getAppClass('system');
+    system.waitForStartup();
+    home.waitForLaunch();
+  });
+
+  test('apps grid is LTR in LTR language', function() {
+    var communicationManifestUrl =
+        'app://communications.gaiamobile.org/manifest.webapp';
+    var phoneIcon = home.getIcon(communicationManifestUrl, 'dialer').location();
+    var contactsIcon = home.getIcon(communicationManifestUrl, 'contacts')
+      .location();
+    var messageIcon = home.getIcon('app://sms.gaiamobile.org/manifest.webapp')
+      .location();
```

Comment:
Do we not care about the alignment though? You'll have to excuse if this is a basic question, I don't have much RTL experience - I'd have expected if things are going from right to left that they also align to the right - if it ordered RTL, but aligned to the left would it not feel a bit odd? Given your patch does this, I think it'd be sensible to test it - but if you disagree, I'd be fine with that.